src/Components/Server/src/PublicAPI.Unshipped.txt

```diff
@@ -1,4 +1,5 @@
 #nullable enable
+Microsoft.AspNetCore.Components.Server.CircuitOptions.JavaScriptInitializers.get -> System.Collections.Generic.IList<string!>!
```

Comment:
How come this is part of the public API? I thought the list of initializers was determined by the presence of `.js` files on disk. Under what circumstances would a developer mutate the list at runtime by setting things in these options?

Reply:
I can make this an implementation detail, I think it might be ok if we let folks tweak the list of things that we will autoload.

Reply:
I don't actually mind which way you go with this, but it still appears to be an open question.

Reply:
Let's discuss this in this afternoon's standup and make a decision on it.